### ISSUE TITLE
When running the "plan" on 1-org there are a number of warnings flagging potential issues downstream

### ISSUE THREAD

**Issue by mromascanu123** [terraform-google-modules/terraform-example-foundation#1188]
### TL;DR

When running ./tf-wrapper plan production on 1-org here are some of the errors and warnings we observe in the log. Can't tell if benign or "by design" but it's worrying
=====================
2024-04-11T13:12:51.304Z [ERROR] AttachSchemaTransformer: No provider config schema available for provider["terraform.io/builtin/terraform"]
=====================
2024-04-11T13:13:38.320Z [DEBUG] provider.terraform-provider-google_v4.84.0_x5: 2024/04/11 13:13:38 [DEBUG] Read bucket bkt-prj-c-logging-3n5z-org-logs-a0f7 at location https://www.googleapis.com/storage/v1/b/bkt-prj-c-logging-3n5z-org-logs-a0f7
2024-04-11T13:13:38.321Z [DEBUG] provider.terraform-provider-google_v4.84.0_x5
2024-04-11T13:13:38.330Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_storage[0].google_storage_bucket.bucket, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .requester_pays: planned value cty.False for a non-computed attribute
      - .default_event_based_hold: planned value cty.False for a non-computed attribute
=============
2024-04-11T13:13:38.419Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_pubsub[0].google_service_account.pubsub_subscriber[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .description: planned value cty.StringVal("") for a non-computed attribute
      - .disabled: planned value cty.False for a non-computed attribute
==============
2024-04-11T13:13:38.435Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_pubsub[0].google_pubsub_subscription.pubsub_subscription[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .filter: planned value cty.StringVal("") for a non-computed attribute
      - .retain_acked_messages: planned value cty.False for a non-computed attribute
      - .enable_exactly_once_delivery: planned value cty.False for a non-computed attribute
      - .enable_message_ordering: planned value cty.False for a non-computed attribute
      - .message_retention_duration: planned value cty.StringVal("604800s") for a non-computed attribute
      - .expiration_policy: block count in plan (1) disagrees with count in config (0)
=============
2024-04-11T13:13:38.515Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_logbucket[0].google_logging_project_bucket_config.bucket, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .locked: planned value cty.False for a non-computed attribute
===========
2024-04-11T13:13:38.625Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_lbk"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .description: planned value cty.StringVal("") for a non-computed attribute
2024-04-11T13:13:38.629Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_pub"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .description: planned value cty.StringVal("") for a non-computed attribute
2024-04-11T13:13:38.630Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_sto"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .description: planned value cty.StringVal("") for a non-computed attribute
=============

2024-04-11T13:13:39.151Z [WARN]  Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_logbucket[0].google_logging_linked_dataset.linked_dataset[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .bigquery_dataset: block count in plan (1) disagrees with count in config (0)

### Expected behavior

Clean execution of "plan" phase

### Observed behavior

Errors and warnings which might be environment-related or code-related, hard to say

### Terraform Configuration

```hcl
Nothing special (see attached screenshot)
```

### Terraform Version

```sh
$ terraform version
Terraform v1.6.0
on linux_amd64
```

### Additional information

![terraform_tfvars](https://github.com/terraform-google-modules/terraform-example-foundation/assets/162719251/9f91c1b7-f592-499c-9711-2ec59e0a71ae)

**Comment by daniel-cit** [comment]
- 2024-04-11T13:12:51.304Z [ERROR] AttachSchemaTransformer: No provider config schema available for provider["terraform.io/builtin/terraform"]

See [No provider config schema available message for builtin provider should not be labelled ERROR in output](https://github.com/hashicorp/terraform/issues/34207)


- 2024-04-11T13:13:38.330Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_storage[0].google_storage_bucket.bucket, but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .requester_pays: planned value cty.False for a non-computed attribute
  - .default_event_based_hold: planned value cty.False for a non-computed attribute
- 2024-04-11T13:13:38.419Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_pubsub[0].google_service_account.pubsub_subscriber[0], but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .description: planned value cty.StringVal("") for a non-computed attribute
  - .disabled: planned value cty.False for a non-computed attribute
- 2024-04-11T13:13:38.435Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_pubsub[0].google_pubsub_subscription.pubsub_subscription[0], but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .filter: planned value cty.StringVal("") for a non-computed attribute
  - .retain_acked_messages: planned value cty.False for a non-computed attribute
  - .enable_exactly_once_delivery: planned value cty.False for a non-computed attribute
  - .enable_message_ordering: planned value cty.False for a non-computed attribute
  - .message_retention_duration: planned value cty.StringVal("604800s") for a non-computed attribute
  - .expiration_policy: block count in plan (1) disagrees with count in config (0)
- 2024-04-11T13:13:38.515Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.destination_logbucket[0].google_logging_project_bucket_config.bucket, but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .locked: planned value cty.False for a non-computed attribute
- 2024-04-11T13:13:38.625Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_lbk"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .description: planned value cty.StringVal("") for a non-computed attribute
- 2024-04-11T13:13:38.629Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_pub"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .description: planned value cty.StringVal("") for a non-computed attribute
- 2024-04-11T13:13:38.630Z [WARN] Provider "registry.terraform.io/hashicorp/google" produced an invalid plan for module.logs_export.module.log_export["1025416132306_sto"].google_logging_folder_sink.sink[0], but we are tolerating it because it is using the legacy plugin SDK.
The following problems may be the cause of any confusing errors from downstream operations:
  - .description: planned value cty.StringVal("") for a non-computed attribute

See [Context around the log entry “…tolerating it because it is using the legacy plugin SDK.”](https://discuss.hashicorp.com/t/context-around-the-log-entry-tolerating-it-because-it-is-using-the-legacy-plugin-sdk/1630)

**Comment by eeaton** [comment]
Thanks for the investigation Daniel. To the best of my understanding, it looks like the root cause is upstream in the Google provider. 

From [another thread](https://discuss.hashicorp.com/t/tolerating-it-because-the-provider-is-using-the-legacy-plugin-sdk/50356/3) I found the following. (It's about the AWS provider, but I expect the explanation is likely consistent with the Google provider)
